pydeseq2/dds.py

```diff
@@ -88,63 +98,49 @@ class DeseqDataSet:
 
     Attributes
     ----------
-    design_matrix : pandas.DataFrame
-        A DataFrame with experiment design information (to split cohorts).
-        Indexed by sample barcodes. Unexpanded, *with* intercept.
-
-    n_processes : int
-        Number of cpus to use for multiprocessing.
-
-    size_factors : pandas.Series
-        DESeq normalization factors.
-
-    non_zero_genes : pandas.Index
-        Index of genes that have non-uniformly zero counts.
-
-    genewise_dispersions : pandas.Series
-        Initial estimates of gene counts dispersions.
+    X
+        A ‘number of samples’ x ‘number of genes’ count data matrix.
 
-    trend_coeffs : pandas.Series
-        Coefficients of the trend curve: :math:`f(\mu) = \alpha_1/ \mu + a_0`.
+    obs
```

Comment:
I understand that `obs` is a dictionary, what is the type of each of its values? an `nd.array`? `pandas.Series`?

Reply:
cf above the [anndata doc](https://anndata.readthedocs.io/en/latest/generated/anndata.AnnData.html#anndata.AnnData)